test/patterns/send_null/test_send_null.py

```diff
@@ -0,0 +1,33 @@
+import os
+from unittest import TestCase
+from aibolit.patterns.send_null.send_null import SendNull
+from pathlib import Path
+
+
+class TestSendNull(TestCase):
+    dir_path = Path(os.path.realpath(__file__)).parent
+    method_send_null_finder = SendNull()
+
+    def test_one_send(self):
+        lines = self.method_send_null_finder.value(Path(self.dir_path, 'BaseKeyframeAnimation.java'))
+        self.assertEqual(lines, [146])
+
+    def test_multi_level_invocation(self):
+        lines = self.method_send_null_finder.value(Path(self.dir_path, 'Configuration.java'))
+        self.assertEqual(lines, [442, 379, 638, 656, 830, 866, 1362, 2393, 2988, 3080, 3492])
+
+    def test_no_null_methods(self):
+        lines = self.method_send_null_finder.value(Path(self.dir_path, 'FillContent.java'))
+        self.assertEqual(lines, [])
+
+    def test_simple_invocation(self):
+        lines = self.method_send_null_finder.value(Path(self.dir_path, 'FJIterateTest.java'))
+        self.assertEqual(lines, [493])
```

Comment:
Check the following cases if you didn't test it:

`a.method_call(2, b.method_call(null))`
`a.method_call().method_call(b).method_call(null)`
Check, if we can pass a ternary operator:
`a.method_call(5>8? 5:null)`
`a.method_call(5>8? null:5)`
Check Constructor:
`new Object(null)`

Reply:
Will do, thanks